### PR TITLE
very temp 429 fix

### PR DESCRIPTION
**Pull request by Roki100**
very simple and temp fix for latest update that causes big boi 429, code can be edited
im also not sure if i did it right(implemented)/if it is working but i dont get 429 after adding this for now

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -105,6 +105,10 @@ const downloadFromInfoCallback = (stream, info, options) => {
         Range: `bytes=${options.range.start || '0'}-${options.range.end || ''}`
       });
     }
+     requestOptions.headers = Object.assign({}, requestOptions.headers, {
```
Comment (fent) [inline lib/index.js]: extra space here
```suggestion
    requestOptions.headers = Object.assign({}, requestOptions.headers, {
```

**Comment by fent** [comment]
can others comment on this? does this solve the 429 status error for you?

**Comment by userlip** [comment]
Did not fix it for me

**Comment by RocoGreen** [comment]
Didn't fix for me too

**Comment by Roki100** [comment]
for me it fixed getting 429 from getting audio data

**Comment by omarroth** [comment]
Currently testing this as a fix for omarroth/invidious#811 (omarroth/invidious@dad885c051e7ea7fa0734dac01bf4bdaf8429c07). Additionally added:

```
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36
```

Will report back with results.

**Comment by RocoGreen** [comment]
@Roki100 nope, i didnt get rate limited

**Comment by Roki100** [comment]
@RocoGreen then it works

**Comment by userlip** [comment]
I just tried executing the BasicInfo function on a server (that is 429 blocked). With all 3 Options (normal ytdl-core, using this pull request, using this pull request + @omarroth user agent)

In all 3 cases no output was generated and no error was logged (which I find strange) however the same script gives me a perfectly good output on my local machine and a new server, so I dont think its the script.

If anyone wants me to test something on that server that is blocked, let me know

**Comment by Roki100** [comment]
@userlip this isnt 429 removal pr, it should prevent getting more 429 in the future

**Comment by omarroth** [comment]
After testing the fix for a couple days I have not encountered the issue again. I've been testing on about 15 droplets from DigitalOcean with each droplet handling about 25 req/s.

**Comment by WaqasIbrahim** [comment]
> After testing the fix for a couple days I have not encountered the issue again. I've been testing on about 15 droplets from DigitalOcean with each droplet handling about 25 req/s.

@omarroth Were those droplets rate limited in the past? Are they fresh?

**Comment by omarroth** [comment]
Looks like I spoke a bit too soon. Quite literally 10 minutes after all nodes have been banned.

@WaqasIbrahim They are fresh.

**Comment by Roki100** [comment]
@omarroth are you using other features than downloading like info etc.?

**Comment by omarroth** [comment]
Yes, in addition to video metadata there are also requests for comments, search, channels, playlists, etc. All are made currently with the above headers.

**Comment by HexDevv** [comment]
fixed it for a second and then it broke again, you're getting close

**Pull request closed without merging**